plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -52,6 +52,14 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
   },
 }));
 
+const TEXT_FONT_SIZE: number = 15;
```

Comment:
The type is redundant here.

Matter of taste, but this could also be collected in an object to have fewer things in the upper scope, and have less uppercase stuff.

```ts
const dimensions = {
  nodeWidth: 180,
  nodeHeight: 90,
  ...
```

Reply:
created the object to hold more information about all the dimensions